vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -107,14 +107,14 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 				return nil, errors.New(msg)
 			}
 
-         if errno == syscall.ECONNRESET {
-            msg += fmt.Sprintf(" retryCount=%d Retrying Request\n", retryCount)
-            log.Warnf(msg)
-            time.Sleep(3 * time.Millisecond)
-         } else {
-            log.Errorf(msg)
-            return nil, errors.New(msg)
-         }
+			if errno == syscall.ECONNRESET || errno == syscall.ETIMEDOUT {
+				msg += fmt.Sprintf(" retryCount=%d Retrying Request\n", retryCount)
```

Comment:
nit - define a function to validate if given error is among errors identified as retryable.